src/instructlab/model/serve.py

```diff
@@ -76,6 +76,11 @@
         " Example: --vllm-args='--dtype=auto' --vllm-args='--enable-lora'"
     ),
 )
+@click.option(
+    "--vllm-background",
```

Comment:
@danmcp @leseb what do you think of the name for flag for `ilab model serve`? This was just my first attempt at a name.

Reply:
Also do you think both this that the default should be drawn from the config like the `max-ctx-size` option is above? The default in that case would be set to true, which is not what we want.

Reply:
> @danmcp @leseb what do you think of the name for flag for `ilab model serve`? This was just my first attempt at a name.

My personal preference is to not put vllm in the name.  The intention being it would apply to any serving where background vs foreground was an option.  The help can explain which ones it applies to.



> Also do you think both this that the default should be drawn from the config like the `max-ctx-size` option is above? The default in that case would be set to true, which is not what we want.

My preference would be this is not a config option.  It's an option for what the command should show you vs config about a user's environment/model/hardware.  Meaning it can easily change from use case to use case.  So having a default in config can be confusing.  For example, if my config is set to false and I tell someone else to run it expecting particular output, they may experience something different with their config.  So the rule I am suggesting is command behaviors are not config.  You could for example write a command as ilab model background-serve (not suggesting we do this) and is would make logical sense.

Reply:
You make a great point about this not being a config option. I've scaled down the size and scope of the PR and in the process of doing so I removed the config option.